api/controllers/arithmeticController.js

```diff
@@ -11,7 +11,7 @@ exports.calculate = function(req, res) {
   });
 
   var operations = {
-    'add':      function(a,b) { return +a + +b },
+    'add':      function(a,b) { return a + b },
```

Comment:
```suggestion
    // Using + operator to type cast variables as integers in order to prevent string concatenation
    'add':      function(a,b) { return +a + +b },
```